foundationdb/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,39 @@
+name: FoundationDB
+files:
+- name: foundationdb.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/default
```

Comment:
Let's put this after check-specific options

Reply:
Not sure I follow; you mean that this line (with the `template` key) should move to the end of the file?

Reply:
Yes

```yaml

  - template: instances
    options:
    - name: cluster_file
      description: |
        A path to the cluster file specifing connection to the FoundationDB.
        If it is not passed, the check will rely on the binding library to find
        the file using standard locations (e.g. `/etc/foundationdb/fdb.cluster` or
        `/etc/foundationdb/fdb.cluster`).
      value:
        example: /var/lib/foundationdb/fdb.cluster
        type: string
    ...
    - template: instances/default
```

Reply:
Thanks; done.